atom/browser/ui/win/notify_icon.cc

```diff
@@ -49,7 +49,18 @@ void NotifyIcon::HandleClickEvent(const gfx::Point& cursor_pos,
                                   bool left_mouse_click) {
   // Pass to the observer if appropriate.
   if (left_mouse_click) {
-    NotifyClicked();
+    NOTIFYICONIDENTIFIER icon_id;
+    memset(&icon_id, 0, sizeof(NOTIFYICONIDENTIFIER));
+    icon_id.uID = icon_id_;
+    icon_id.hWnd = window_;
+    icon_id.cbSize = sizeof(NOTIFYICONIDENTIFIER);
+
+    RECT rect;
+    Shell_NotifyIconGetRect(&icon_id, &rect);
+
+    int width = rect.right - rect.left;
+    int height = rect.bottom - rect.top;
+    NotifyClicked(gfx::Rect(rect.left, rect.top, width, height));
```

Comment:
Oh, that is quite handy :+1: . I didn't see this because of the whole `#if defined(OS_WIN)` crapp surrounding it :).

Anyhow, it seems to work.